odysseus/halow_setup/install/nrc-wizard

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Is this script made by you?

Reply:
Yes, everything in /install/ is me and /source/ is newracom or ALFA.  I will describe all of this in the README.